pages/index.vue

```diff
@@ -185,7 +204,7 @@ export default {
 }
 </script>
 
-<style lang="postcss" scoped>
```

Comment:
Why removed?

Reply:
Sorry, it was a mistake during testing, but it has now been restored.